src/main/java/gregtech/api/metatileentity/multiblock/RecipeMapMultiblockController.java

```diff
@@ -140,6 +146,19 @@ private void resetTileAbilities() {
         this.energyContainer = new EnergyContainerList(Lists.newArrayList());
     }
 
+    protected IMultipleTankHandler extendedImportFluidList(IMultipleTankHandler fluids) {
+        List<IFluidTank> tanks = new ArrayList<>(fluids.getFluidTanks());
+        for (var handler : getAbilities(MultiblockAbility.IMPORT_ITEMS)) {
+            if (handler instanceof IFluidTank tank) {
+                tanks.add(tank);
```

Comment:
Is there any chance that `fluids` would already contain the fluid tanks that are associated with the dual handler? Want to see if we are not adding things if they are already there.

Reply:
this depends on how the dual handler MTE registers its abilities, but i could add a `contains` check here